### ISSUE TITLE
[Bug] Condition creation fails with StaleElementReferenceException

### ISSUE THREAD

**Issue by john-dupuy** [ManageIQ/integration_tests#8103]
Unable to create a condition via `appliance.collections.conditions.create()` method

cf. `cfme/control/explorer/conditions.py`

e.g. [here](https://github.com/ManageIQ/integration_tests/blob/0751cabee21c8ff1b25cc1b87aee21fa981b5ca7/cfme/control/explorer/conditions.py#L237)
``` 
condition = condition_collection.create(
    condition_class,
    fauxfactory.gen_alphanumeric(),
    expression="fill_field({} : Address )".format(
        condition_class.FIELD_VALUE),
    scope="fill_field({} : Name, INCLUDES, {})".format(
        condition_class.FIELD_VALUE, fauxfactory.gen_alpha())
)
```

This will raise `StaleElementReferenceException: Message: Couldn't handle it` and appears to impact CFME 5.9 as well as 5.10. Once at this page, it is not possible to navigate via the Accordion, as it is grayed out. Seems to be impacting several test cases, but is most relevant for tests in:
`cfme/tests/control/test_basic.py` [here](https://github.com/ManageIQ/integration_tests/blob/0751cabee21c8ff1b25cc1b87aee21fa981b5ca7/cfme/tests/control/test_basic.py#L530)

**Comment by mshriver** [comment]
@john-dupuy do any of your open PRs address this? Would you like the issue assigned to you?

**Comment by john-dupuy** [comment]
@mshriver yes to both, this should be fixed by #8133
